### ISSUE TITLE
Browserstack username & key for development

### ISSUE THREAD

**Issue by MarkusPint** [tdeekens/grunt-browsersnapsy#2]
Currently it seems like I would have to manually remove my Browserstack username & access key from /grunt/tasks/browsersnapsy.js before each commit to prevent them from ending up public.

Any ideas how to fix this?

**Comment by tdeekens** [comment]
Fair point. We had the credentials in private repos for a while and sometimes ended up expecting people to have them set as environment variables accessed via `process.env`. After all, when made public those variables can be encrypted using `travis encrypt` in case of a public repo.

Do you have any further ideas we can improve this on?
